5-predict_subtype_reconstructed_data.R

```diff
@@ -74,25 +74,33 @@ for (seed in filename.seeds) {
       
       # read in reconstructed data from current platform and reconstruction 
       # method
-      file.identifier <- paste(plt, rcn, seed, sep = "_")
+      file.identifier <- paste(rcn, plt, seed, sep = "_")
       recon.rds <- recon.files[grep(file.identifier, recon.files)]
       recon.list <- readRDS(recon.rds)
       
       # get confusionMatrix objects
-      plt.list[[rcn]] <- PredictReconDataWrapper(train.list = train.list,
-                                                 recon.list = recon.list,
-                                                 sample.df = sample.df)
+      plt.list[[rcn]] <- PredictWrapper(train.model.list = train.list,
```

Comment:
Ha! Well at least I'm consistent :joy: I do remember now that the lapply loop was tough to figure out. Perhaps calculating it inside the function and returning both arguments would solve both issues? Does this make sense or am I not understanding completely? Happy to chat about it if that would help